generate_wordstr_box.py

```diff
@@ -32,11 +32,13 @@
 # load gt
 with io.open(args.txt, "r", encoding='utf-8') as f:
     lines = f.read().strip().split('\n')
+    if len(lines) > 1:
```

Comment:
As the ground truth text file should contain exactly one line, you might as well check for exactly 1.

Reply:
13a58fb